content/assignments/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Graded Assignments"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 100
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+<!-- TODO: Link to A 0 when it's completed -->
+
+## What is a Graded Assignment?
+Graded Assignments will count towards your final grade in the course. Each assignment is graded pass-fail. While [Assignment 0] (LINK HERE) is more of a tutorial to get you used to autograding in JavaScript, you have to get a passing grade on each assignment in order for you to continue on to the next unit. Each assignment builds on concepts introduced over multiple lessons, so we encourage you to begin reviewing the assignment and working on the different tasks as soon as the assignment becomes available in Canvas.
```

Comment:
Please update link to current assignment 0, make a todo to change after this class when assignment 0 is added to this book.

Reply:
link is updated and pointing to A0 in the sphinx JS book.  TODO added